core/java/com/google/instrumentation/stats/RpcConstants.java

```diff
@@ -218,6 +211,124 @@
           DistributionAggregationDescriptor.create(RPC_BYTES_BUCKET_BOUNDARIES),
           Arrays.asList(RPC_SERVER_METHOD));
 
+  // Interval Stats
+  static final Duration MINUTE = Duration.create(60, 0);
+  static final Duration HOUR = Duration.create(60 * 60, 0);
+
+  // Census defined rpc client {@link IntervalViewDescriptor}s.
+  public static final IntervalViewDescriptor RPC_CLIENT_ROUNDTRIP_LATENCY_INTERVAL_VIEW =
+      IntervalViewDescriptor.create(
+          "rpc client roundtrip_latency",
+          "Latency in msecs",
```

Comment:
I would add mention of minute-hour stats in the description, to avoid confusion with other similar names. Maybe also should be reflected in the name?